examples/package.json

```diff
@@ -33,7 +33,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "lodash-es": "^4.17.21",
-    "near-sdk-js": "file:../"
+    "near-sdk-js": "file:../",
+    "yargs": "^17.5.1"
```

Comment:
Looks like this change should not be here.

Reply:
Right! Have removed, ty.